rest-service/manager_rest/test/test_list_v1.py

```diff
@@ -57,6 +57,12 @@ def instance(*args, **kwargs):
             # restore original function
             config.instance = original_instance_func
 
+    def test_insecure_endpoints_enabled_by_default(self):
+        try:
+            self.client.events.get(execution_id='111')
+        except CloudifyClientError, e:
+            self.assertNotEquals(405, e.status_code)
```

Comment:
i'd also assert the code is of the 200 family, to make sure we didn't get any 404, 400 or 50x before the 405 could even get launched or something like that